{{cookiecutter.app_name}}/Makefile

```diff
@@ -1,19 +1,39 @@
-.PHONY: env-setup env-teardown dev install-dependencies test
+.PHONY: env-setup env-teardown db/migrate db/rollback migratoin/create migration/status dev install-dependencies test
 
 env-setup:
 	docker-compose -f docker-compose.dev.yml up -d
 
 env-teardown:
 	docker-compose -f docker-compose.dev.yml down
 
-dev: env-setup
+db/migrate:
+	make check-env
+	goose -dir database/migrations -table "migration_versions" postgres "postgres://postgres:postgres@0.0.0.0:${DB_PORT}/test-gin-template_${ENV}?sslmode=disable" up
+
+db/rollback:
+	goose -dir database/migrations -table "migration_versions" postgres "postgres://postgres:postgres@0.0.0.0:$(DB_PORT)/test-gin-template_${ENV}?sslmode=disable" down
```

Comment:
@olivierobert According to slack conversation. I decided to add make command to wrap goose command. Goose migration command needs a database URL in order to connect to the database and migrate it. In this case I need ENV which I can pass from make test and make dev command but I think DB PORT should come from config.

I'm still struggling with how to implement this properly.